src/pconn.cc

```diff
@@ -295,6 +295,26 @@ IdleConnList::findUseable(const Comm::ConnectionPointer &aKey)
     return Comm::ConnectionPointer();
 }
 
+void
+IdleConnList::closeAllTo(const CachePeer * const p)
+{
+    for (auto right = size_; right > 0; --right) {
+        const auto i = right - 1;
+        const auto conn = theList_[i];
+
+        if (!Comm::IsConnOpen(conn))
+            continue;
```

Comment:
They probably do not exist in IdleConnList, but if they do (and this `if` statement implies that they might), then we do not want to leave closed connections to removed peers. The method is called closeAllTo() to keep the symmetry with existing closeN(), but both should not check Comm::IsConnOpen() AFAICT. The existing closeN() method does not.

```suggestion
```

P.S. closeN() does not check `conn` for being nil either.

Reply:
Done.